cni/network/network.go

```diff
@@ -591,3 +596,147 @@ func (plugin *netPlugin) Delete(args *cniSkel.CmdArgs) error {
 
 	return nil
 }
+
+// Update handles CNI update commands.
+// Update is only supported for multitenancy and to update routes.
```

Comment:
If this is only for multitenancy, can we move this to cni/network/multitenancy.go

Reply:
the plan is for this to handle any update..